### PR TITLE
Spring cleaning

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: scripts/jenkins/jobs-obs/openstack-trackupstream.yml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+            - tripleo-heat-templates
+            - diskimage-builder
+      - axis:
+          type: slave
+          name: slave
+          values:
+            - cloud-trackupstream
+    execution-strategy:
+      combination-filter: |
+        ! ( [ "Cloud:OpenStack:Liberty:Staging", "Cloud:OpenStack:Kilo:Staging", "Cloud:OpenStack:Juno:Staging"].contains(project) && [ "openstack-neutron-lbaas", "openstack-neutron-fwaas", "openstack-neutron-vpnaas", "openstack-tuskar", "openstack-tuskar_ui", "python-oslo.messaging", "python-oslosphinx", "python-oslo.rootwrap", "python-openstackclient", "python-keystonemiddleware", "python-tuskarclient", "python-troveclient", "tripleo-image-elements", "tripleo-heat-templates", "diskimage-builder", "openstack-zaqar", "python-ironicclient", "python-zaqarclient", "python-barbicanclient", "python-os-apply-config", "python-os-cloud-config", "python-os-collect-config"].contains(component)       ||
```
Comment (toabctl) [inline scripts/jenkins/jobs-obs/openstack-trackupstream.yml]: NIT: this could be formatted a bit more readable.

**Comment by toabctl** [comment]
+1

**Comment by toabctl** [comment]
It's already winter, but anyway +1

**Comment by toabctl** [comment]
@bmwiedemann @jdsn opinions?

**Comment by bmwiedemann** [comment]
:+1:  (assuming that it works)

**Pull request merged**